contracts/rust/src/bin/diff_test.rs

```diff
@@ -395,12 +395,17 @@ fn main() {
             println!("args: {:?}", cli.args);
         }
         Action::GenClientWallet => {
-            let mut rng = jf_utils::test_rng();
-
-            if cli.args.len() != 1 {
-                panic!("Should provide arg1=senderAddress");
+            if cli.args.len() != 2 {
+                panic!("Should provide arg1=senderAddress arg2=seed");
             }
 
+            // Use seed from cli to generate different bls keys
+            let seed_value: u8 = cli.args[1].parse::<u8>().unwrap();
+            let mut seed_bytes_length_32 = [0u8; 32];
+            seed_bytes_length_32[0] = seed_value;
```

Comment:
super nit: why don't we simply: 
`let seed = [seed_value; 32]`?

Reply:
Done in d7bc82e.